build/squire-raw.js

```diff
@@ -2288,21 +2288,20 @@ var onPaste = function ( event ) {
             }
         }
 
-        if ( plainItem ) {
-            item.getAsString( function ( text ) {
-                self.insertPlainText( text, true );
-            });
-        }
-
         if ( rtfItem ) {
             item.getAsString( function ( text ) {
-                if ( !text ) {
-                    // RTF items without text may contain OLE embedded image data.
-                    // Firing willPaste to notify consumers an image may have been pasted.
-                    this.fireEvent.call( this, 'willPaste', imagePasteEvent );
+                pasteEvent['isImage'] = !text;
```

Comment:
I think this might come from a merge, but we should probably not get this property by string.